From 0757a5d521a172011ab30eb3f581dd18f813833b Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Tue, 4 Oct 2016 11:43:47 +0200
Subject: Use the Debian-provided libjbig.

---
 Makefile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 38d65e1..5b9eba5 100644
--- a/Makefile
+++ b/Makefile
@@ -125,10 +125,6 @@ FILES	=	\
 		Makefile \
 		foo2zjs.c \
 		foo2zjs.1in \
-		jbig.c \
-		jbig.h \
-		jbig_ar.c \
-		jbig_ar.h \
 		zjsdecode.c \
 		zjsdecode.1in \
 		zjs.h \
@@ -275,7 +271,7 @@ MANPAGES+=	foo2hbpl2-wrapper.1 foo2hbpl2.1 hbpldecode.1
 MANPAGES+=	gipddecode.1
 MANPAGES+=	foo2zjs-pstops.1 arm2hpdl.1 usb_printerid.1
 MANPAGES+=	printer-profile.1
-LIBJBG	=	jbig.o jbig_ar.o
+LIBJBG =       -ljbig
 BINPROGS=
 
 ifeq ($(UNAME),Linux)
