From def34908383ca355f85cbb847e6befc3c41ebc9f Mon Sep 17 00:00:00 2001
From: Helmut Grohne <helmut@subdivi.de>
Date: Fri, 5 Oct 2018 16:33:27 +0200
Subject: Don't test for stdio.h in fullpath

Closes: #910322
---
 Makefile | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/Makefile b/Makefile
index cfc5d9a..2aba8d2 100644
--- a/Makefile
+++ b/Makefile
@@ -399,15 +399,6 @@ all-test:
 	    echo "      ***"; \
 	    exit 1; \
 	fi
-	@if ! test -f /usr/include/stdio.h; then \
-	    echo "      ***"; \
-	    echo "      *** Error: /usr/include/stdio.h is not installed!"; \
-	    echo "      ***"; \
-	    echo "      *** Install Software Development (gcc) package"; \
-	    echo "      *** for Ubuntu: sudo apt-get install build-essential"; \
-	    echo "      ***"; \
-	    exit 1; \
-	fi
 	@if ! type gs >/dev/null 2>&1; then \
 	    echo "      ***"; \
 	    echo "      *** Error: gs is not installed!"; \
