Improve getweb also for installing the firmware
--- foo2zjs-20090908dfsg.orig/getweb.in
+++ foo2zjs-20090908dfsg/getweb.in
@@ -2,7 +2,7 @@
 
 #
 # (c) Copyright Rick Richardson 2008
-#
+# This version has been modified by Debian
 
 #
 # PLEASE don't remove the copyright statements.  They should 
@@ -12,6 +12,13 @@
 VERSION='$Id: getweb.in,v 1.89 2009/06/23 18:31:20 rick Exp $'
 
 WGETOPTS="--passive-ftp -q"
+ARM2HPDL="/usr/bin/arm2hpdl"
+
+if [ $(id -ru) -ne 0 ]; then
+    echo "You need to be root"
+    exit 1
+fi
+
 
 usage() {
 cat <<EOF
@@ -94,7 +101,7 @@
 }
 
 SYSTEM=`uname -s`
-WGET=`type wget 2>/dev/null`
+WGET=`which wget`
 if [ "$SYSTEM" = Darwin -a "$WGET" = "" ]
 then
     WGETOPTS=
@@ -104,6 +111,13 @@
     }
 fi
 
+putfw() {
+    inputname=$1
+    outputname=$2
+    $ARM2HPDL /usr/share/foo2zjs/tmp/$inputname > /usr/share/foo2zjs/firmware/$outputname
+    rm -f /usr/share/foo2zjs/tmp/*.img
+}
+
 #
 #	Download a .EXE file from the web, unzip it, and extract the
 #	files we want
@@ -126,10 +140,16 @@
     file="$2"
     what="$3"
     
-    wget $WGETOPTS -O $file "$url/$file" ||
-	error "Couldn't download $url/$file"
-    gunzip <$file | tar xvf - $what
-    rm $file
+    if ! [ -d /tmp/foo2zjs ] ; then
+	wget $WGETOPTS -O /usr/share/foo2zjs/tmp/$file "$url/$file" ||
+            error "Couldn't download $url/$file"
+	cd /usr/share/foo2zjs/tmp && gunzip </usr/share/foo2zjs/tmp/$file | tar xvf - $what
+	CHECK=`ls /usr/share/foo2zjs/tmp/*.icm 2>/dev/null`
+	if [ -n "$CHECK" ] ; then
+            mv /usr/share/foo2zjs/tmp/*.icm /usr/share/foo2zjs/icm/
+	fi
+	rm /usr/share/foo2zjs/tmp/$file
+    fi
 }
 
 copyright() {
@@ -150,6 +170,7 @@
 	    #lj1488en.exe \
 	    #sihp1000.img
 	copyright "(c) Copyright Hewlett-Packard 2001"
+	putfw sihp1000.img sihp1000.dl
 	;;
     lj1005|1005)
 	gettgz \
@@ -161,24 +182,28 @@
 	    #lj1005hostbased-en.exe \
 	    #sihp1005.img
 	copyright "(c) Copyright Hewlett-Packard 2002"
+	putfw sihp1005.img sihp1005.dl
 	;;
     lj1018|1018)
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihp1018.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2005"
+	putfw sihp1018.img sihp1018.dl
 	;;
     lj1020|1020)
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihp1020.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2005"
+	putfw sihp1020.img sihp1020.dl
 	;;
     [pP]100[57])
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihpP1005.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2008"
+	putfw sihpP1005.img sihpP1005.dl
 	;;
     [pP]100[68])
 	gettgz \
@@ -203,6 +228,7 @@
 		"*.icm"
 	fi
 	copyright "(c) Copyright Minolta-QMS 1999"
+	putfw sihp1020.img sihp1020.dl
 	;;
     2300dl|2300)
 	if false; then
