Improve getweb also for installing the firmware
--- foo2zjs_20100728dfsg.orig/getweb.in
+++ foo2zjs_20100728dfsg/getweb.in
@@ -2,7 +2,7 @@
 
 #
 # (c) Copyright Rick Richardson 2008
-#
+# This version has been modified by Debian
 
 #
 # PLEASE don't remove the copyright statements.  They should 
@@ -12,6 +12,13 @@
 VERSION='$Id: getweb.in,v 1.91 2010/06/15 16:50:56 rick Exp $'
 
 WGETOPTS="--passive-ftp -q"
+ARM2HPDL="/usr/bin/arm2hpdl"
+
+if [ $(id -ru) -ne 0 ]; then
+    echo "You need to be root"
+    exit 1
+fi
+
 
 usage() {
 cat <<EOF
@@ -95,7 +102,7 @@
 }
 
 SYSTEM=`uname -s`
-WGET=`type wget 2>/dev/null`
+WGET=`which wget`
 if [ "$SYSTEM" = Darwin -a "$WGET" = "" ]
 then
     WGETOPTS=
@@ -105,6 +112,18 @@
     }
 fi
 
+putfw() {
+    inputname=$1
+    outputname=$2
+    $ARM2HPDL $inputname > /usr/share/foo2zjs/firmware/$outputname
+    rm $inputname
+}
+
+puticm() {
+    drivername=$1
+    mv *.icm /usr/share/"$drivername"/icm/
+}
+
 #
 #	Download a .EXE file from the web, unzip it, and extract the
 #	files we want
@@ -151,6 +170,7 @@
 	    #lj1488en.exe \
 	    #sihp1000.img
 	copyright "(c) Copyright Hewlett-Packard 2001"
+	putfw sihp1000.img sihp1000.dl
 	;;
     lj1005|1005)
 	gettgz \
@@ -162,36 +182,42 @@
 	    #lj1005hostbased-en.exe \
 	    #sihp1005.img
 	copyright "(c) Copyright Hewlett-Packard 2002"
+	putfw sihp1005.img sihp1005.dl
 	;;
     lj1018|1018)
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihp1018.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2005"
+	putfw sihp1018.img sihp1018.dl
 	;;
     lj1020|1020)
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihp1020.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2005"
+	putfw sihp1020.img sihp1020.dl
 	;;
     [pP]100[57])
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihpP1005.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
+	putfw sihpP1005.img sihpP1005.dl
 	;;
     [pP]100[68])
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihpP1006.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
+	putfw sihpP1006.img sihpP1006.dl
 	;;
     [pP]1505)
 	gettgz \
 	    http://foo2zjs.rkkda.com/firmware sihpP1505.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
+	putfw sihpP1505.img sihpP1505.dl
 	;;
     2200dl|2200)
 	if false; then
@@ -204,6 +230,7 @@
 		"*.icm"
 	fi
 	copyright "(c) Copyright Minolta-QMS 1999"
+	puticm foo2zjs
 	;;
     2300dl|2300)
 	if false; then
@@ -219,6 +246,7 @@
 	    http://foo2zjs.rkkda.com/icm km2430.tar.gz \
 	    ""
 	copyright "(c) Copyright Minolta-QMS 2001"
+	puticm foo2zjs
 	;;
     2300dl_fw)
 	# 2300DL firmware upgrade to v2.55
@@ -247,12 +275,14 @@
 	    done
 	fi
 	copyright "(c) Copyright Minolta-QMS 1998"
+	puticm foo2zjs
 	;;
     2430)
 	gettgz \
 	    http://foo2zjs.rkkda.com/icm km2430.tar.gz \
 	    ""
 	copyright "(c) Copyright Konica Minolta 2003"
+	puticm foo2zjs
 	;;
     2530|2490|6115|2480|1600w|1680|1690|4690|110)
 	gettgz \
@@ -264,8 +294,9 @@
 	    http://foo2lava.rkkda.com/icm km-1600.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2009.  All Rights Reserved"
+	puticm foo2lava
 	;;
-    1215|1500|1600|2600*)
+    1500)
 	gettgz \
 	    http://foo2hp.rkkda.com/icm hpclj2600n.tar.gz \
 	    ""
@@ -278,6 +309,22 @@
 	    http://foo2hp.rkkda.com/icm hp1215.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2008.  All Rights Reserved"
+	puticm foo2oak
+	;;
+    1215|1600|2600*)
+	gettgz \
+	    http://foo2hp.rkkda.com/icm hpclj2600n.tar.gz \
+	    ""
+	gettgz \
+	    http://foo2zjs.rkkda.com/icm km2430.tar.gz \
+	    "km2430_2.icm"
+	copyright "(c) Copyright Hewlett-Packard 2003"
+	copyright "(c) Copyright Rick Richardson 2006.  All Rights Reserved"
+	gettgz \
+	    http://foo2hp.rkkda.com/icm hp1215.tar.gz \
+	    ""
+	copyright "(c) Copyright Rick Richardson 2008.  All Rights Reserved"
+	puticm foo2hp
 	;;
     300|310|315|2160|3160|3175|6110)
 	gettgz \
@@ -288,6 +335,7 @@
 	    http://foo2qpdl.rkkda.com/icm samclp315.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2008.  All Rights Reserved"
+	puticm foo2qpdl
 	;;
     600|610)
 	;;
@@ -297,6 +345,7 @@
 	    http://foo2slx.rkkda.com/icm lexc500.tar.gz \
 	    ""
 	copyright "(c) Copyright Lexmark 2006"
+	puticm foo2slx
 	;;
 
     oki)
@@ -312,24 +361,28 @@
 	    mv "$base.ICM" "$base.icm"
 	done
 	copyright "(c) Copyright Okidata 2005, 2006"
+	puticm foo2hiperc
 	;;
     3100|3200|5100|5150|5200|5250)
 	gettgz \
 	    http://foo2hiperc.rkkda.com/icm okic3200.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2005"
+	puticm foo2hiperc
 	;;
     3300|3400|3530)
 	gettgz \
 	    http://foo2hiperc.rkkda.com/icm okic3400.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2006"
+	puticm foo2hiperc
 	;;
     5500|5600|5800)
 	gettgz \
 	    http://foo2hiperc.rkkda.com/icm okic5600.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2006"
+	puticm foo2hiperc
 	;;
 
     xPPD)
@@ -415,6 +468,10 @@
     usage
 fi
 
+OLDPWD=`pwd`
+mkdir -p /tmp/foo2zjs
+cd /tmp/foo2zjs
+
 for i in $*
 do
     case "$1" in
@@ -433,6 +490,8 @@
 	# getone 2430 already done
 	# getone cpwl # file doesn't exist 1/28/08
 
+	getone 1500
+
 	getone 2600n
 	# getone 1215 # already done
 
@@ -453,3 +512,5 @@
 	;;
     esac
 done
+
+cd "$OLDPWD"
