Fixes errors in manpages
--- a/foo2hp.1in
+++ b/foo2hp.1in
@@ -198,7 +198,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHP}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2qpdl-wrapper.1in
+++ b/foo2qpdl-wrapper.1in
@@ -225,7 +225,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLQPDL}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/arm2hpdl.1in
+++ b/arm2hpdl.1in
@@ -38,7 +38,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2hp2600-wrapper.1in
+++ b/foo2hp2600-wrapper.1in
@@ -235,7 +235,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHP}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/hipercdecode.1in
+++ b/hipercdecode.1in
@@ -163,7 +163,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHC}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2hiperc-wrapper.1in
+++ b/foo2hiperc-wrapper.1in
@@ -205,7 +205,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHC}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2xqx.1in
+++ b/foo2xqx.1in
@@ -161,7 +161,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLXQX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/qpdldecode.1in
+++ b/qpdldecode.1in
@@ -126,7 +126,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLQPDL}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/usb_printerid.1in
+++ b/usb_printerid.1in
@@ -44,7 +44,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2hiperc.1in
+++ b/foo2hiperc.1in
@@ -173,7 +173,4 @@
 Rick Richardson <rick.richardson@comcast.com>
 .br
 ${URLHC}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/oakdecode.1in
+++ b/oakdecode.1in
@@ -92,7 +92,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLOAK}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2oak.1in
+++ b/foo2oak.1in
@@ -194,7 +194,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLOAK}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/zjsdecode.1in
+++ b/zjsdecode.1in
@@ -99,7 +99,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2zjs-pstops.1in
+++ b/foo2zjs-pstops.1in
@@ -48,7 +48,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2zjs.1in
+++ b/foo2zjs.1in
@@ -228,7 +228,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2qpdl.1in
+++ b/foo2qpdl.1in
@@ -192,7 +192,4 @@
 Rick Richardson <rick.richardson@comcast.com>
 .br
 ${URLQPDL}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2oak-wrapper.1in
+++ b/foo2oak-wrapper.1in
@@ -221,7 +221,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLOAK}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2zjs-wrapper.1in
+++ b/foo2zjs-wrapper.1in
@@ -266,7 +266,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/xqxdecode.1in
+++ b/xqxdecode.1in
@@ -105,7 +105,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLXQX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2slx-wrapper.1in
+++ b/foo2slx-wrapper.1in
@@ -190,7 +190,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLSLX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2xqx-wrapper.1in
+++ b/foo2xqx-wrapper.1in
@@ -162,7 +162,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLXQX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2lava-wrapper.1in
+++ b/foo2lava-wrapper.1in
@@ -229,7 +229,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2slx.1in
+++ b/foo2slx.1in
@@ -157,7 +157,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLSLX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/foo2lava.1in
+++ b/foo2lava.1in
@@ -187,7 +187,4 @@
 Rick Richardson <rick.richardson@comcast.com>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/slxdecode.1in
+++ b/slxdecode.1in
@@ -103,7 +103,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLSLX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/lavadecode.1in
+++ b/lavadecode.1in
@@ -96,7 +96,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLLAVA}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/opldecode.1in
+++ b/opldecode.1in
@@ -115,7 +115,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLRO}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- a/printer-profile.1in
+++ b/printer-profile.1in
@@ -57,7 +57,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
