Fixes errors in manpages
--- foo2zjs_20100728working.orig/foo2hp.1in
+++ foo2zjs_20100728working/foo2hp.1in
@@ -198,7 +198,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHP}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2qpdl-wrapper.1in
+++ foo2zjs_20100728working/foo2qpdl-wrapper.1in
@@ -225,7 +225,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLQPDL}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/arm2hpdl.1in
+++ foo2zjs_20100728working/arm2hpdl.1in
@@ -38,7 +38,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2hp2600-wrapper.1in
+++ foo2zjs_20100728working/foo2hp2600-wrapper.1in
@@ -235,7 +235,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHP}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/hipercdecode.1in
+++ foo2zjs_20100728working/hipercdecode.1in
@@ -162,7 +162,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHC}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2hiperc-wrapper.1in
+++ foo2zjs_20100728working/foo2hiperc-wrapper.1in
@@ -205,7 +205,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLHC}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2xqx.1in
+++ foo2zjs_20100728working/foo2xqx.1in
@@ -161,7 +161,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLXQX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/qpdldecode.1in
+++ foo2zjs_20100728working/qpdldecode.1in
@@ -126,7 +126,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLQPDL}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/usb_printerid.1in
+++ foo2zjs_20100728working/usb_printerid.1in
@@ -44,7 +44,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2hiperc.1in
+++ foo2zjs_20100728working/foo2hiperc.1in
@@ -173,7 +173,4 @@
 Rick Richardson <rick.richardson@comcast.com>
 .br
 ${URLHC}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/oakdecode.1in
+++ foo2zjs_20100728working/oakdecode.1in
@@ -92,7 +92,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLOAK}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2oak.1in
+++ foo2zjs_20100728working/foo2oak.1in
@@ -194,7 +194,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLOAK}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/zjsdecode.1in
+++ foo2zjs_20100728working/zjsdecode.1in
@@ -99,7 +99,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2zjs-pstops.1in
+++ foo2zjs_20100728working/foo2zjs-pstops.1in
@@ -45,7 +45,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2zjs.1in
+++ foo2zjs_20100728working/foo2zjs.1in
@@ -224,7 +224,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2qpdl.1in
+++ foo2zjs_20100728working/foo2qpdl.1in
@@ -192,7 +192,4 @@
 Rick Richardson <rick.richardson@comcast.com>
 .br
 ${URLQPDL}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2oak-wrapper.1in
+++ foo2zjs_20100728working/foo2oak-wrapper.1in
@@ -221,7 +221,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLOAK}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2zjs-wrapper.1in
+++ foo2zjs_20100728working/foo2zjs-wrapper.1in
@@ -262,7 +262,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/xqxdecode.1in
+++ foo2zjs_20100728working/xqxdecode.1in
@@ -105,7 +105,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLXQX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2slx-wrapper.1in
+++ foo2zjs_20100728working/foo2slx-wrapper.1in
@@ -190,7 +190,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLSLX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2xqx-wrapper.1in
+++ foo2zjs_20100728working/foo2xqx-wrapper.1in
@@ -162,7 +162,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLXQX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2lava-wrapper.1in
+++ foo2zjs_20100728working/foo2lava-wrapper.1in
@@ -229,7 +229,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2slx.1in
+++ foo2zjs_20100728working/foo2slx.1in
@@ -157,7 +157,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLSLX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/foo2lava.1in
+++ foo2zjs_20100728working/foo2lava.1in
@@ -187,7 +187,4 @@
 Rick Richardson <rick.richardson@comcast.com>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/slxdecode.1in
+++ foo2zjs_20100728working/slxdecode.1in
@@ -103,7 +103,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLSLX}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/lavadecode.1in
+++ foo2zjs_20100728working/lavadecode.1in
@@ -96,7 +96,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLLAVA}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/opldecode.1in
+++ foo2zjs_20100728working/opldecode.1in
@@ -115,7 +115,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLRO}/
-'/"
-'/"
-'/"
 .em pdf_outline
--- foo2zjs_20100728working.orig/printer-profile.1in
+++ foo2zjs_20100728working/printer-profile.1in
@@ -55,7 +55,4 @@
 Rick Richardson <rick.richardson@comcast.net>
 .br
 ${URLZJS}/
-'/"
-'/"
-'/"
 .em pdf_outline
